website/src/components/communitySpotlightCard/styles.module.css

```diff
@@ -79,6 +79,7 @@ div.spotlightMemberCard {
   }
   div.spotlightMemberCard {
     margin-bottom: 0;
+    padding-left: 0 !important;
```

Comment:
Is there any easy way to make this more specific so that we don't need to use the dreaded `!important`? Not a big deal if not.

Reply:
Just updated!